showcase/app/templates/layouts/app-frame/frameless/demo-full-app-frame-with-app-header.hbs

```diff
@@ -256,7 +256,6 @@
 
   <Frame.Footer>
     <Hds::AppFooter as |AF|>
-      {{! @glint-expect-error - TODO: investigate why this line fails }}
```

Comment:
@alex-ju haha fixing the empty class components to templateOnly components revealed an issue with how the AppFooter component imported the types, which I fixed and then I think that fixed the weird typing issue with these files! Who woulda thought.